frontend/src/components/RegistrationView/FormFields.tsx

```diff
@@ -50,109 +22,51 @@ export const InputField: React.FC<FieldProps & InputProps> = ({
     return (
         <div>
             <Label>{title}</Label>
-            <Input placeholder={placeholder} {...field} {...props} onBlur={handleBlur}/>
+            <Input placeholder={placeholder} {...field} {...props} onBlur={handleBlur} />
             {isTouched &&
             error && <ErrorMessage>{error}</ErrorMessage>}
         </div>
     );
 }
 
+interface RadioItem {
+    value: string,
+    label: string
+}
 
-const Select = styled.div`
-    width: 100%;
-    display: flex;
-    padding: 1em 0 0.5em;
-`;
-
-const Radio = styled.input.attrs({type: 'radio'})`
-    &:checked, &:not(:checked) {
-        position: absolute;
-        visibility: hidden;
-    }
-
-    &:checked + label, &:not(:checked) + label {
-        position: relative;
-        padding-left: 26px;
-        cursor: pointer;
-        color: #000;
-        line-height: 1.5em;
-        height: 100%;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-    }
-
-    &:checked + label:before, &:not(:checked) + label:before {
-        content: '';
-        position: absolute;
-        left: 0;
-        width: 20px;
-        height: 20px;
-        border: 1px solid #000;
-        border-radius: 100%;
-        background: #fff;
-    }
-    &:checked + label:after, &:not(:checked) + label:after {
-        content: '';
-        width: 14px;
-        height: 14px;
-        background: #3c8fde;
-        top: calc(50% - 7px);
-        position: absolute;
-        left: 4px;
-        border-radius: 100%;
-    }
-    &:not(:checked) + label:after {
-        opacity: 0;
-    }
-`;
-
-const SelectOption = styled.div`
-    display: inline-block;
-    margin-right: 0.3em;
-`;
-
-const SelectLabel = styled.label`
-    display: inline-block;
-    font-size: 0.8em;
-`;
-
-const Description = styled.div`
-    font-size: 0.8em;
-    white-space: pre-wrap;
-`
-
+interface RadioProps {
+    data: RadioItem,
+}
 
-const RadioField: React.FC<InputProps & FieldProps> = ({
+const RadioField: React.FC<RadioProps & FieldProps> = ({
     field,
-    title,
+    data,
     form,
     ...props
   }) => {
     return (
         <SelectOption>
-            <Radio type="radio" id={title} 
+            <Radio type="radio" id={data.value} 
```

Comment:
Is it good that id comes from value? I think it's changing value? Now if its changing value, and 2 fields value is same --> have same id --> same htmlFor --> clicking one fields label targets both of the fields, or otherway acts weird.

Reply:
it's a radio button, the values are fixed and different

Reply:
I mean it's not a value for the whole field, it's a value of each radio option